.env

```diff
@@ -0,0 +1,11 @@
+RABBIT_HOST=rabbit
```

Comment:
Not sure if this file is needed at all. From my perspective, it's better to specify default values right in the application code. Let me know if you have a different vision.
But in general, you can leave it like that.

Reply:
This file was taken from pycharm. It doesn't needed.